values.yaml

```diff
@@ -1178,6 +1177,88 @@ ingressGateways:
   gateways:
     - name: ingress-gateway
 
+# Configuration options for terminating gateways. Default values for all
+# terminating gateways are defined in `terminatingGateways.defaults`. Any of
+# these values may be overridden in `terminatingGateways.gateways` for a
+# specific gateway with the exception of annotations. Annotations will
+# include both the default annotations and any additional ones defined
+# for a specific gateway.
+# Requirements: consul >= 1.8.0 and consul-k8s >= 0.16.0 if using
+# global.acls.manageSystemACLs.
+terminatingGateways:
+  # Enable terminating gateway deployment. Requires `connectInject.enabled=true`.
```

Comment:
Should we inlcude `client.enabled` and `client.grpc` as a requirement as well?

Reply:
Added `client.enabled`, but `client.rpc` is enabled by default so leaving it off.